external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -485,33 +465,6 @@ function findTracedFunctionsFromPath(pkgRoot: string, pkgModules: string[]): str
     }).flat();
 }
 
-/**
- * Find all functions that have a corresponding trace file by looking at
- * the content of the trace file and its name (`<package>__<module>__<function_name>.json`).
- * We need to match the package address, module name, and function name in the trace
- * file itself as this is the only place where we can find the (potentially matching)
- * package address (module name and function name could be extracted from the trace
- * file name).
- *
- * @param pkgRoot root directory of the package.
- * @param pkgAddr package address.
- * @param module module name.
- * @returns list of functions of the form `<package>::<module>::<function_name>`.
- * @throws Error (containing a descriptive message) if no traced functions are found for the package.
- */
-function findTracedFunctionsFromTrace(pkgRoot: string, pkgAddr: number, module: string): string[] {
-    const filePaths = getTraceFiles(pkgRoot);
-    const result: string[] = [];
-    for (const p of filePaths) {
-        const tracePath = path.join(pkgRoot, 'traces', p);
-        const tracedFunctionInfo = getTracedFunctionInfo(tracePath);
```

Comment:
This is the part where we would have to decompress a trace file, which we do not do here as I failed to update this in the recent [PR](https://github.com/MystenLabs/sui/pull/21906)